random/provider.go

```diff
@@ -15,6 +15,7 @@ func Provider() terraform.ResourceProvider {
 			"random_shuffle": resourceShuffle(),
 			"random_pet":     resourcePet(),
 			"random_string":  resourceString(),
+			"random_range":   resourceRange(),
```

Comment:
Naming things is always tricky... when I first saw this name I expected that the result would be a random start/end segment of a list, or some such thing, since "range" makes me think of a sub-portion of a list.

What do you think about calling this `random_integer`, as a sibling of `random_string`?

Reply:
I had a similar thought. However, `random_integer` gives me the impression that it does not necessarily require a range but allows me to get any integer. I could also make `min` and `max` optional and set it to `min=0` and `max=INTEGER_MAX`. 🧐